perception/point_cloud_to_pose_system.py

```diff
@@ -0,0 +1,245 @@
+import numpy as np
+import yaml
+
+from pydrake.systems.framework import AbstractValue, LeafSystem
+from pydrake.util.eigen_geometry import Isometry3, AngleAxis
```

Comment:
`eigen_geometry` has been moved to `pydrake.common.eigen_geometry`. `pydrake.util.eigen_geometry` might have been deprecated.